applications/nrf5340_audio/src/modules/led.c

```diff
@@ -184,6 +184,10 @@ static int led_device_tree_parse(void)
 				return ret;
 			}
 
+		} else if (strstr(led_labels[i], "LED_MONO_BLUE") && CONFIG_AUDIO_DEV == HEADSET) {
+			/* Not initialize LED_MONO_BLUE here on headset devices. 
```

Comment:
When commenting on multiple lines we use this format:


```suggestion
			/* Not initialize LED_MONO_BLUE here on headset devices.
			 * On headsets this LED is controlled by the audio sync timer.
			 */
```